openshift_sti_images/python.adoc

```diff
@@ -0,0 +1,191 @@
+= Python
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+The OpenShift https://github.com/openshift/sti-python/tree/master/3.3[Python] image repository contains the sources and Dockerfiles for building various versions of Python platform images. The resulting images can be run either by Docker or using https://github.com/openshift/source-to-image[STI].
+
+== Versions
+Currently, OpenShift only supports version https://github.com/openshift/sti-python/tree/master/3.3[3.3] of Python.
```

Comment:
we should strike this wording from our other docs too...  "openshift provides a 3.3 version" would be better.